src/Header.js

```diff
@@ -48,7 +55,9 @@ class Header extends Component {
 			<AppBar
 				title={<span>{this.props.title}</span>}
 				iconElementLeft={iconElementLeft}
-				iconElementRight={<IconButton href="/settings"><IconSettings/></IconButton>}
+				iconElementRight={
+					iconElementRight ? iconElementRight : <IconButton href="/settings"><IconSettings/></IconButton>
```

Comment:
Why don't you just set a default value in line 27 when initialising `iconElementRight`? This would get overridden in case another or no icon is needed and would avoid this not-so-nice inline `if`.